queries/lua/indents.scm

```diff
@@ -6,6 +6,7 @@
   (for_statement)
   (for_in_statement)
   (repeat_statement)
+  (return_statement)
```

Comment:
also adding `(variable_declaration)` and `(field)` have a nice effect.

Reply:
adding `(arguments)` has a nice effect as well.

Reply:
Also `local_variable_declaration`